test/stats/with-non-asset-asset/a-clone.js

```diff
@@ -0,0 +1 @@
+module.exports = 'module a';
```

Comment:
Why do we need this file?

Reply:
Not necessary.

Using `CopyWebpackPlugin` could cause `hidden assets` added in stats.json.

 This file is a copied asset generated by a test webpack build.

I can delete this file if it's wanted.

Reply:
Yep, delete it please.

Reply:
Done